### ISSUE TITLE
Dark line between modeline and echo area

### ISSUE THREAD

**Issue by rjww** [hlissner/emacs-doom-theme#31]
Wondering if there's a way to get rid of this. [Here's a question I opened on StackExchange.](http://emacs.stackexchange.com/questions/29873/whats-this-line-between-the-modeline-and-the-echo-area)

**Comment by hlissner** [comment]
If you're using my emacs.d, changes are you're also using `window-divider-mode`, which adds this border. You can either turn off the bottom border with `(setq window-divider-default-bottom-width 0)` or turn off `window-divider-mode`.

**Comment by hlissner** [comment]
For reference, here is my window-divider config:

```emacs-lisp
;; `window-divider-mode' gives us finer control over the border between windows.
;; The native border "consumes" a pixel of the fringe on righter-most splits (in
;; Yamamoto's emacs-mac at least), window-divider does not.
;; NOTE Only available on Emacs 25.1+
(when (boundp 'window-divider-mode)
  (setq window-divider-default-places t
        window-divider-default-bottom-width 1
        window-divider-default-right-width 0)
  (window-divider-mode +1))
```

**Comment by rjww** [comment]
Brilliant, thanks. I've pulled a lot of stuff from your config, and that was one of the pieces.
